app/components/action_button.rb

```diff
@@ -1,14 +1,14 @@
 # frozen_string_literal: true
 
-# Draws a blue button in the side menu
+# Draws a red button on the item detail page
```

Comment:
Haha! `s/blue/red/` 😆 

Given we will be changing these buttons' colors over time, I imagine, I wonder if we can drop the color altogether.

Reply:
Removed color